zuul.d/projects.yaml

```diff
@@ -417,6 +417,72 @@
       jobs:
         - ansible-tox-linters
 
+- project:
+    name: github.com/ansible/molecule
+    check: &cfg-molecule
+      jobs:
+        - name: lint
```

Comment:
lint is too generic, can we make this molecule-lint or molcule-tox-linters?  As long as we prefix with molecule- that is the main issue. This is because jobs names need to be unique in tenants.

Reply:
Also, lets move job stanza into top-level for ansible-zuul-jobs, as they are untrusted and can do pre-merge testing on changes.